### ISSUE TITLE
Tooltip error: Multi chart in on same page shows tooltip only for last created chart

### ISSUE THREAD

**Issue by 5ri** [areski/python-nvd3#107]
When i try to use two charts of the same time the tooltip for the previous charts disappears.
Think it was a NDV3 issue as shown here: https://github.com/novus/nvd3/issues/587

Is it a matter of rebuilding with the latest ndv3.js ?

**Comment by areski** [comment]
yes but there is important changes in 1.8 so that might not be straightforward but it might be nice to start the work on this. I would appreciate a PR :)

**Comment by SaptadeepaRC** [comment]
I had an issue .don't know how to post individually. I am using nvd3 package on python but the code from nvd3 import multichart doesn't work. I want to print two bars against the same x axis along with 2corresponding lines. Lineplusbar is leading to overlapping of bars. All I want is a proper union of multiplebarchart and multipleline chart. Please help
